tools/test_translation_manager.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import unittest
+
+from translation_manager import *
+
+class TestTranslationManager(unittest.TestCase):
+
+    def test_copy_new_keys_to_locale(self):
+        self.assertEqual('foo'.upper(), 'FOO')
```

Comment:
Nit: Is this leftover from making the tests?

Reply:
yes i missed those